app/DoctrineMigrations/Version20181001160533.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
`showCustomPopupMessage` og eventuelle andre ikke-nullable felter må legges til for gamle data i DB-migrasjonen.